yorkie/src/main/kotlin/dev/yorkie/document/time/TimeTicket.kt

```diff
@@ -9,13 +9,39 @@ internal data class TimeTicket(
     val actorID: ActorID,
 ) : Comparable<TimeTicket> {
 
+    fun toIDString(): String {
```

Comment:
I don't think we need `toIDString` since we are using TimeTicket instance as key